.moai/specs/SPEC-ZONE-REGISTRY-HARDEN-001/acceptance.md

```
@@ -0,0 +1,126 @@
+- **Given** M2 완료 후 트리
+- **When**:
+  ```bash
+  go test -run TestRegistryTupleDigestRejectsSubstitution -v ./internal/constitution/
+  ```
+- **Then** `PASS` — 서브테스트가 개수 보존 변이 4종(ID 문자 삽입 치환 / `zone:` 값 치환 / `zone_class:` 값 치환 / `canary_gate:` 반전) 각각에 대해 산출 digest ≠ `wantTupleDigest`를 단정하고 전부 통과. RED 근거(구현 전 동일 변이가 가드를 통과함을 보인 출력)는 run-phase §E.2에 원문 보존
+
+### AC-ZRH-006 — validator·CLI·임베드 축
+
+- **Given** M1+M2 완료 후 트리
+- **When**:
+  ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)